packages/core/database/src/schema/builder.ts

```diff
@@ -259,30 +302,21 @@ const createHelpers = (db: Database) => {
         dropForeignKey(tableBuilder, updatedForeignKey.object);
       }
 
-      // for mysql only, dropForeignKey also removes the index, so don't drop it twice
-      const isMySQL = db.config.connection.client === 'mysql';
-      const ignoreForeignKeyNames = isMySQL
```

Comment:
I'm still investigating why this was necessary before, now that I've discovered multiple dropIndexes are fine. I might need to put this back.

Reply:
I'm pretty sure it is necessary